Dockerfile

```diff
@@ -56,7 +56,7 @@ COPY ./migrations /backend/migrations
 
 EXPOSE 8888
 ENTRYPOINT /docker-entrypoint.sh $0 $@
-CMD ["uvicorn", "--reload", "--host=0.0.0.0", "--port=8888", "main:app"]
+CMD ["uvicorn", "--reload", "--host=0.0.0.0", "--port=8888", "server.main:app"]
```

Comment:
I take it that this was adjusted so that it could work from devbuoyant? Or is it because the path is server/server/main.py?

Reply:
The latter, but the path is actually "server/main.py" relative to the location of this Dockerfile. This is an important distinction as the pathing is referenced in the [context of the location of this Dockerfile](https://github.com/clairBuoyant/devBuoyant/pull/3/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R48), which is why "server" is not repeated twice.

This was just a bug that flew under the radar. With this fix in place, I am able to remove `command: python server/main.py` from the "docker-compose.yaml" file [here](https://github.com/clairBuoyant/devBuoyant/pull/3/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3L50).

Reply:
Cool! Good to know!